plonky2/src/plonk/circuit_builder.rs

```diff
@@ -410,23 +485,21 @@ impl<F: RichField + Extendable<D>, const D: usize> CircuitBuilder<F, D> {
         );
     }
 
+    /// Adds a gate type to the set of gates to be used in this circuit. This can be useful
+    /// in conditional recursion to uniformize the gates set of the different circuits.
```

Comment:
It took me couple of seconds, because I kept reading it as the gates being set instead of the set of gates.
```suggestion
    /// in conditional recursion to uniformize the set of gates of the different circuits.
```